ci/rpm/install-base-el8.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh -e
+
+# Install OS updates and packages as required for building DAOS on EL 8 and
+# derivatives.  Include basic tools and daos dependencies that come from the core repos.
+# This is for a daos test install environment, not a build environment.
+
+# This script use used by docker but can be invoked from elsewhere, in order to run it
+# interactively then these two commands can be used to set dnf into automatic mode.
+# dnf --assumeyes install dnf-plugins-core
+# dnf config-manager --save --setopt=assumeyes=True
+
+dnf --nodocs install \
```

Comment:
Not familiar with that code.   Perhaps a future enhancement.
Just discovered that the "git" package is needed on all Dockerimages used in CI.

Reply:
I'm not familiar with it either but I know it exists and using it would be preferable to adding our dependency list to yet another place in the repo.

The code I'm thinking about is at https://github.com/daos-stack/daos/blob/master/utils/ansible/ftest/tasks/el8/devs_dependencies.yml#L13 and it installs build-deps but if there is a way of telling dnf to install the packages from the spec file then you should use it.

Reply:
I'm slightly surprised there doesn't seem to be a similar command to install the run-time dependencies.  As there isn't anything obvious then a list here will suffice.

What is git used for, that seems a bit surprising.

Reply:
I think it is some code in the pipeline-lib runTest.sh that can probably be removed as it is duplicating the commit pragma code.   If git is not present in the docker container, it generates noise in the log.